test/cql/lwt_test.result

```diff
@@ -2045,7 +2045,11 @@ Error from server: code=2200 [Invalid query] message="Cannot store NULL in list
 +---------+
 > -- negative list index
 > update lwt set c = [true, false] where a = 1 if c[-1] = false;
-Error from server: code=2200 [Invalid query] message="Invalid negative list index -1"
++-------------+---------+
+| [applied]   | c       |
+|-------------+---------|
+| False       | [False] |
++-------------+---------+
```

Comment:
Nitpick: to check if c[-1] is null, maybe it's better to check "c[-1] is null" instead of c[-1] = false?
(but I don't know, is IS NOT implemented in LWT? maybe it's not implemented).

Reply:
IS NULL does't exist in LWT yet.